composer.json

```diff
@@ -31,31 +31,36 @@
         "ext-pdo_mysql": "*",
         "ext-xml": "*",
         "ext-zip": "*",
-        "beberlei/assert": "^3.2",
-        "browscap/browscap": "6.0.39",
+        "browscap/browscap": "^6.0",
```

Comment:
Note, I've proposed changing to using semver, hopefully Dependabot will automatically update us when we release `browscap/browscap` now... so that'll mostly automate the second half of the release process (apart from clicking a few buttons to merge the PR if build passes)!